lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -1,234 +1,168 @@
 .course-index {
```

Comment:
It looks like there's a lot here, but most of it is removing the jQuery UI style hooks. jQuery UI adds a bunch of classes for accordions that these styles were hooking into. Since I removed jQuery UI, I ended up writing styles for more basic things and our custom classes, also utilizing the new markup/structure. Does this clear things up?